### PR TITLE
#789 -- Fixes link without href does not get highlighted in high cont…

### PR DESCRIPTION
**Pull request by aleksandrjPersonal**
…rast

#### Pull request checklist

- [x] Addresses an existing issue: #789
- [x] Include a change request file if publishing 
- [ ] New feature, bugfix, or enhancement
  - [ ] Includes tests
- [ ] Documentation update

#### Description of changes

Adds a and href to button to allow for the correct coloring in HighContrast mode

#### Focus areas to test

Link component

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/Link/Link.tsx [hunk 1]
```diff
@@ -48,7 +48,15 @@ export class Link extends BaseComponent<ILinkProps, any> implements ILink {
             onClick={ this._onClick }
             ref={ this._resolveRef('_link') }
           >
-            { children }
+            <a href=""
```
Comment (dzearing) [inline packages/office-ui-fabric-react/src/components/Link/Link.tsx]: A link without an href should be rendered as a button. You can't render an A tag in a button...

**Comment by dzearing** [comment]
When you file PRs, please fill out the PR form above.

File: packages/office-ui-fabric-react/src/components/Link/Link.tsx (hunk 1 above)
Comment (aleksandrjPersonal) [inline packages/office-ui-fabric-react/src/components/Link/Link.tsx]: The desired color is automagically applied to the href. Will keep looking for a way to trigger the desired color for the button.

**Comment by aleksandrjPersonal** [comment]
@dzearing Who would know if this was a real issue or not?

**Comment by micahgodbolt** [comment]
@aleksandrjPersonal the original issue was concerned with buttons looking like something other than text in high contrast mode, right? Isn't this an issue of styles in high contrast mode?

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt The original request as I understand it is that we want the our text in the button to look and act indistinguishable from a <a href='' /> tag.

In high contrast mode, 6 theme colors are applied to screen elements. (Text, Hyperlinks, Disabled Text, Selected Text, Button Text, Background) the value of these 6 colors change per theme, and are also customizable for users at any time. 

So it's not about simply applying a purple color when in HighContrast Mode, it's about applying a 'mystery token' that would resolve to the correct color as the theme changes. Something like '$-ms-color-current-highcontrast-hyperlink' -- I would guess. 

Additionally the browser does not expose the named hooks it has into these system colors. It only displays what the calculated RGB value is, but not the token that it used to resolve that RGB value.

So it appears we are *not* going to be able to style a non-hyperlink to look like a hyperlink without these named system hooks.

****************************************

I talked it over with @mikewheaton and he noted that since the button is clearly visible in black and white HC mode, these buttons are probably fine as is. And this bug can be removed as a 'won't fix'

Thoughts?

**Comment by aleksandrjPersonal** [comment]
@micahgodbolt @battletoilet Closing as a won't fix, by design.

**Pull request closed without merging**